### PR TITLE
Cleanup

### PR DESCRIPTION
**Pull request by uhoh-itsmaciek**
* Use built-in CLI features where possible
 * Move from underscore to lodash since CLI uses lodash
 * Use https://github.com/feross/standard for formatting since the CLI uses it

### CODE REVIEW

File: commands/clusters.js [hunk 1]
```diff
@@ -258,28 +257,17 @@ HerokuKafkaClusters.prototype.addonForSingleClusterCommand = function* (cluster)
 };
 
 HerokuKafkaClusters.prototype.findByClusterName = function (addons, cluster) {
-  return addons.kafkas.filter(function (addon) { return _.contains(addon.config_vars, cluster) || addon.name == cluster; });
-};
-
-HerokuKafkaClusters.prototype.checkConfirmation = function* (context, message) {
```
Comment (idan) [inline commands/clusters.js]: Where'd this go? We don't need it anymore?

Comment (uhoh-itsmaciek) [inline commands/clusters.js]: There's a built-in [`cli.confirm`](https://github.com/heroku/heroku-cli-util#confirm-app) we can use now.

File: commands/configure_topic.js [hunk 2]
```diff
@@ -1,40 +1,12 @@
...
-          cli.error(`could not parse retention time '${value}'`);
-          process.exit(1);
-        }
-        value = parsed;
-      }
-      out[flag.name] = value;
-    }
-  });
-  return out;
-}
```
Comment (uhoh-itsmaciek) [inline commands/configure_topic.js]: The CLI now checks that no extraneous flags have been passed to the command.

File: commands/shared.js [hunk 3]
```diff
@@ -1,21 +1,21 @@
...
 
-const _ = require('underscore');
+const _ = require('lodash');
 
 function clusterConfig(addon, config) {
   if (!addon) {
     return null;
   }
-  let urlVar = _.find(addon.config_vars, function(key) {
+  let urlVar = _.find(addon.config_vars, function(val, key) {
```
Comment (uhoh-itsmaciek) [inline commands/shared.js]: #thankslodash

**Comment by idan** [comment]
LGTM! 👍

**Comment by uhoh-itsmaciek** [comment]
I tested this pretty thoroughly (went through all the commands), and fixed a bunch of bugs and other rough edges (mostly missing newlines after some output). I'll ship after demo day.

**Pull request merged**